src/vos/vos_pool.c

```diff
@@ -318,6 +320,33 @@ vos_wal_metrics_init(struct vos_wal_metrics *vw_metrics, const char *path, int t
 	if (rc)
 		D_WARN("Failed to create WAL waiters telemetry: "DF_RC"\n", DP_RC(rc));
 
+	for (i = 0; i < D_TM_IO_LAT_BUCKETS_NR; i++) {
```

Comment:
Adding single vwm_wal_lat should be sufficient, looks D_TM_DURATION is a more proper telemetry type.

Reply:
- [ ] Use `D_TM_DURATION`  instead of `D_TM_STATS_GAUGE` telemetry type

Reply:
> * [x]  Use `D_TM_DURATION`  instead of `D_TM_STATS_GAUGE` telemetry type

Fixed with commit cb986f734